cardano-cli/src/Cardano/CLI/Byron/Tx.hs

```diff
@@ -215,7 +215,7 @@ txSpendUTxOByronPBFT nId sk txIns outs = do
                      TxMintNone
                      TxScriptValidityNone
                      TxGovernanceActionsNone
-                     TxVotesNone
+                     Nothing
```

Comment:
This is the only downside. We should use the type's records to make this more obvious.

Reply:
Yep.  Will do this in a future PR.